pkg/cdi/cache_test.go

```diff
@@ -1634,6 +1637,215 @@ containerEdits:
 	}
 }
 
+func TestCacheTransientSpecs(t *testing.T) {
+	type testCase struct {
+		name         string
+		specs        []string
+		invalid      map[int]bool
+		expected     [][]string
+		numSpecFiles []int
+	}
+	for _, tc := range []*testCase{
+		{
+			name: "invalid spec",
+			specs: []string{
+				`
+cdiVersion: "0.3.0"
```

Comment:
Question: SHould we use the latests spec version for tests like this?

Reply:
Good point. I updated it like that.